sdcm/tester.py

```diff
@@ -367,6 +368,9 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
         if self.params.get("use_ldap"):
             self._init_ldap()
 
+        self._init_data_validation()
+        self.partitions_dict_before = None
+        self.partitions_attributes = None
```

Comment:
should we only call these 3 lines `if self.params.get('data_validation'):` ?

Reply:
its possible, but then we'll have to add this condition of `if self.params.get('data_validation'):` in several places instead.

Reply:
fixed